src/commons/nativeJS/EvalErrorLocator.ts

```diff
@@ -0,0 +1,86 @@
+import { SourceError } from 'js-slang/dist/types';
```

Comment:
Aw man, this whole file feels so fragile and hacky..

But there's probably no real better way